<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 61);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="WelcomePage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 47, 48);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="WelcomeWidget" name="welcomeWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DatabasePage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="DatabaseWidget" name="databaseWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UnlockPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="UnlockWidget" name="unlockWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CreateDbPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="createDBwidget" name="createWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="EntryEditPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="EditEntry" name="editEntryWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="EditGroupPage">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="EditGroup" name="editGroupWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PasswordGeneratorPage">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>237</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PasswordGenereatorWidget" name="widget" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>236</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 48);
color: rgb(216, 216, 216);
selection-background-color: rgb(0, 85, 0);
</string>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionNew_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Database"/>
    <addaction name="actionSave_Database_As"/>
    <addaction name="separator"/>
    <addaction name="actionUpload_To_Dropbox"/>
    <addaction name="actionDownload_From_Dropbox"/>
    <addaction name="actionSync_With_Dropbox"/>
    <addaction name="separator"/>
    <addaction name="actionLock_Database"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEntries">
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>Entries</string>
    </property>
    <addaction name="actionNew_Entry"/>
    <addaction name="actionEdit_Entry"/>
    <addaction name="actionDelete_Entry"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Username"/>
    <addaction name="actionCopy_Password"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Notes"/>
    <addaction name="actionCopy_Title"/>
    <addaction name="actionCopy_Url"/>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>Groups</string>
    </property>
    <addaction name="actionNew_Group"/>
    <addaction name="actionEdit_Group"/>
    <addaction name="actionDelete_Group"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPassword_Generetor"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuEntries"/>
   <addaction name="menuGroups"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(59, 59, 61);
</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 48);
border-color: rgb(46, 47, 48);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Database"/>
   <addaction name="actionSave_Database"/>
   <addaction name="actionSave_Database_As"/>
   <addaction name="actionLock_Database"/>
   <addaction name="actionClose_Database"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Group"/>
   <addaction name="actionEdit_Group"/>
   <addaction name="actionDelete_Group"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Entry"/>
   <addaction name="actionEdit_Entry"/>
   <addaction name="actionDelete_Entry"/>
   <addaction name="separator"/>
   <addaction name="actionCopy_Username"/>
   <addaction name="actionCopy_Password"/>
   <addaction name="actionCopy_Notes"/>
   <addaction name="actionCopy_Title"/>
   <addaction name="actionCopy_Url"/>
   <addaction name="separator"/>
   <addaction name="actionPassword_Generetor"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionNew_Database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/add.png</normaloff>:/Icons/add.png</iconset>
   </property>
   <property name="text">
    <string>New  Database...</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/folder.png</normaloff>:/MenuIcons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Database...</string>
   </property>
  </action>
  <action name="actionSave_Database_As">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/download.png</normaloff>:/MenuIcons/download.png</iconset>
   </property>
   <property name="text">
    <string>Save Database As</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/floppy.png</normaloff>:/MenuIcons/floppy.png</iconset>
   </property>
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
  <action name="actionClose_Database">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/close.png</normaloff>:/MenuIcons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Database</string>
   </property>
  </action>
  <action name="actionLock_Database">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/padlock.png</normaloff>:/MenuIcons/padlock.png</iconset>
   </property>
   <property name="text">
    <string>Lock Database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/sign-out.png</normaloff>:/MenuIcons/sign-out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/add (1).png</normaloff>:/MenuIcons/add (1).png</iconset>
   </property>
   <property name="text">
    <string>New Entry...</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/edit (1).png</normaloff>:/MenuIcons/edit (1).png</iconset>
   </property>
   <property name="text">
    <string>Edit Entry...</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/x-mark.png</normaloff>:/MenuIcons/x-mark.png</iconset>
   </property>
   <property name="text">
    <string>Delete Entry...</string>
   </property>
  </action>
  <action name="actionCopy_Username">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/id-card.png</normaloff>:/MenuIcons/id-card.png</iconset>
   </property>
   <property name="text">
    <string>Copy Username</string>
   </property>
  </action>
  <action name="actionCopy_Password">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/key.png</normaloff>:/MenuIcons/key.png</iconset>
   </property>
   <property name="text">
    <string>Copy Password</string>
   </property>
  </action>
  <action name="actionCopy_Notes">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/notes.png</normaloff>:/MenuIcons/notes.png</iconset>
   </property>
   <property name="text">
    <string>Copy Notes</string>
   </property>
  </action>
  <action name="actionCopy_Title">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/title.png</normaloff>:/MenuIcons/title.png</iconset>
   </property>
   <property name="text">
    <string>Copy Title</string>
   </property>
  </action>
  <action name="actionCopy_Url">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/web.png</normaloff>:/MenuIcons/web.png</iconset>
   </property>
   <property name="text">
    <string>Copy Url</string>
   </property>
  </action>
  <action name="actionNew_Group">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/add (2).png</normaloff>:/MenuIcons/add (2).png</iconset>
   </property>
   <property name="text">
    <string>New Group...</string>
   </property>
  </action>
  <action name="actionEdit_Group">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/writing.png</normaloff>:/MenuIcons/writing.png</iconset>
   </property>
   <property name="text">
    <string>Edit Group...</string>
   </property>
  </action>
  <action name="actionDelete_Group">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/file.png</normaloff>:/MenuIcons/file.png</iconset>
   </property>
   <property name="text">
    <string>Delete Group...</string>
   </property>
  </action>
  <action name="actionPassword_Generetor">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/die.png</normaloff>:/MenuIcons/die.png</iconset>
   </property>
   <property name="text">
    <string>Password Generetor</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/MenuIcons/setting.png</normaloff>:/MenuIcons/setting.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpload_To_Dropbox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/dropbox_upload.png</normaloff>:/Icons/dropbox_upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload To Dropbox</string>
   </property>
  </action>
  <action name="actionDownload_From_Dropbox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/dropbox_download.png</normaloff>:/Icons/dropbox_download.png</iconset>
   </property>
   <property name="text">
    <string>Download From Dropbox</string>
   </property>
  </action>
  <action name="actionSync_With_Dropbox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/cloud.png</normaloff>:/Icons/cloud.png</iconset>
   </property>
   <property name="text">
    <string>Sync With Dropbox</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WelcomeWidget</class>
   <extends>QWidget</extends>
   <header>welcomewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UnlockWidget</class>
   <extends>QWidget</extends>
   <header>unlockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>createDBwidget</class>
   <extends>QWidget</extends>
   <header>createdbwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidget</class>
   <extends>QWidget</extends>
   <header>databasewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditEntry</class>
   <extends>QWidget</extends>
   <header>editentry.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditGroup</class>
   <extends>QWidget</extends>
   <header>editgroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PasswordGenereatorWidget</class>
   <extends>QWidget</extends>
   <header>passwordgenereatorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
